ironfish-rust-nodejs/src/frost.rs

```diff
@@ -52,3 +54,60 @@ pub fn round_two(
 
     Ok(bytes_to_hex(&signature_share.serialize()))
 }
+
+#[napi]
+pub struct ParticipantSecret {
+    secret: Secret,
+}
+
+#[napi]
+impl ParticipantSecret {
+    // TODO(hughy): implement Secret ser/de
```

Comment:
new + serialize methods?

Reply:
The `Secret` type in `ironfish-frost` doesn't implement ser/de yet, so new and serialize aren't yet possible